test/spec/GeoExt/component/OverviewMap.test.js

```diff
@@ -342,7 +342,10 @@ describe('GeoExt.component.OverviewMap', function() {
                 overviewMap.recenterParentFromBox();
 
                 var parentCenter = olMap.getView().getCenter();
-                expect(parentCenter).to.eql([1, 1]);
+
+                setTimeout(function() {
+                    expect(parentCenter).to.eql([1, 1]);
+                }, 100);
```

Comment:
Since this is asynchronous now, don't we need to call `done()` manually after `expect(parentCenter).to.eql([1, 1]);`?

The test-case would need adjustment too, of course:

```js
it('creates and adds a Translate interaction', function(done) {
  // …
  setTimeout(function() {
    // …
    done();
  );
});
```